### DISCUSSION TITLE
Will the Anatomy section be a sister site/project or a subsection under the Handbook?

### DISCUSSION THREAD

**Discussion by sugan-reden** [Epicrex/3DArtistsHandbook#8] [Q&A] [answered]
Hi Sidney,

I'll try to move topics that are less issues and more questions/discussions in here hehe.

I saw you had an _Anatomy_ section under the explorer that redirected to another GitHub project.

May I ask your plans with this?
Like the title says, were you planning on making it a sister site or integrating it completely under the 3D Artist's Handbook? Or perhaps something else?

Hope you're doing well,
Cheers 🍻

P.S. I've done some work on my fork, mostly some proofreading stuff, I'll probably be making a pull request soon 👍.

**Comment by sidney-eliot** [comment, marked as answer]
I'm not sure if you already saw what I wrote under the README of the anatomy-for-artist handbook, but I now moved it to the wiki section where the 3D artist handbook links to.

In short, I will most definitely be keeping them separate as it's not only meant for 3D artists, but just as much for 2D / traditional artists as well.

The anatomy for artist wiki started out for me to learn about the structure of the human body. And I think that it succeeded in focusing only on the subset of muscles that are actually important for artist. But I want to make it more digestible and handy for character artists specifically, who think about anatomy very differently than a doctor might. Light taking things like bony landmarks into account or understanding how to overexaggerate parts of a character's anatomy without making it look unbelievable.

Then, at some point, one can also go into creature / monster anatomy (but that is something I don't know too much about currently).

The anatomy handbook is also supposed to be everything related to the body, so stuff like how to create and shape and layer haircuts, for example. This is also why I decided quite a bit back so separate the 3D artist's handbook and the anatomy handbook, as the 3D artist's handbook is a lot more about workflows, software features, techniques and facts while something like anatomy is not some strict ruleset one can follow.

---

I already have a local quartz instance setup for anatomy for artists.

Also, right around that time, the imgur collapse happened, but I thankfully was quick enough and managed to migrate all images to a local image folder, and many of them are now gone on the wiki
![image](https://github.com/user-attachments/assets/4bcb0651-643a-4209-911e-36abb4d98cc2)
as some obviously contained nudity

**Reply by sugan-reden** [reply]
> I'm not sure if you already saw what I wrote under the README of the anatomy-for-artist handbook, but I now moved it to the wiki section where the 3D artist handbook links to.

I did 🙋 . Took me a bit of back-and-forth to realize how it was all set up though x) 

> In short, I will most definitely be keeping them separate as it's not only meant for 3D artists, but just as much for 2D / traditional artists as well.
>
> The anatomy for artist wiki started out for me to learn about the structure of the human body. And I think that it succeeded in focusing only on the subset of muscles that are actually important for artist. But I want to make it more digestible and handy for character artists specifically, who think about anatomy very differently than a doctor might. Light taking things like bony landmarks into account or understanding how to overexaggerate parts of a character's anatomy without making it look unbelievable.
>
> ...
>
> The anatomy handbook is also supposed to be everything related to the body, so stuff like how to create and shape and layer haircuts, for example. This is also why I decided quite a bit back so separate the 3D artist's handbook and the anatomy handbook, as the 3D artist's handbook is a lot more about workflows, software features, techniques and facts while something like anatomy is not some strict ruleset one can follow.

I see, these are good points 👍. I now agree more that this will be better kept separate. :)

> Then, at some point, one can also go into creature / monster anatomy (but that is something I don't know too much about currently).

Sadly same 🫠 

---

> I already have a local quartz instance setup for anatomy for artists.

Oh nice! I also saw some unpublished content on the anatomy-for-astists' `v4` branch, is the local more ahead than this one?
Also, would you be open to pushing it upstream?

> Also, right around that time, the imgur collapse happened, but I thankfully was quick enough and managed to migrate all images to a local image folder, and many of them are now gone on the wiki

Oh yeah I almost forgot about that! 🤯 Good quick thinking you had! 🙌 

> as some obviously contained nudity

Hmm 🤔, curious to know, is that because of GitHub Wikis or the imgur crash? Or something else (i.e. was it a choice or external factor?)

**Reply by sidney-eliot** [reply]
> Hmm 🤔, curious to know, is that because of GitHub Wikis or the imgur crash? Or something else (i.e. was it a choice or external factor?)

Basically, Imgur nuked any images that contained even the smallest amount of suggestive / nsfw content, using AI filters. In addition to that, they set up new rules that auto-deletes images after time if they weren't uploaded with an Imgur account. And a bunch of other stupid things. 

Right before this happened, there were also some archival people who made backups of flagged for deletion images, so if you do ever find dead Imgur links from before 2023 on the internet, then there's a good chance you can still find the image in one of those backup datasets.

**Reply by sugan-reden** [reply]
> Basically, Imgur nuked any images that contained even the smallest amount of suggestive / nsfw content, using AI filters. In addition to that, they set up new rules that auto-deletes images after time if they weren't uploaded with an Imgur account. And a bunch of other stupid things.

Fudge man, that's sad 🫠. I only saw news articles/video titles about this, I didn't know it got up to that point 🫥.

> Right before this happened, there were also some archival people who made backups of flagged for deletion images, so if you do ever find dead Imgur links from before 2023 on the internet, then there's a good chance you can still find the image in one of those backup datasets.

Oh! That's good to know, I'll keep that in mind 👍
